pkg/composer/composer.go

```diff
@@ -41,6 +41,7 @@ type Options struct {
 	EnvFile          string
 	NerdctlCmd       string
 	NerdctlArgs      []string
+	NetworkUsed      func(context.Context, string) (bool, error)
```

Comment:
nit: suggest rename to `NetworkInUse` (similarly `netInUse` below)

Reply:
make sense